app/build.gradle

```diff
@@ -44,5 +44,6 @@ dependencies {
 
     compile "com.google.dagger:dagger:$daggerVersion"
     compile "com.google.dagger:dagger-android:$daggerVersion"
+    compile "com.google.dagger:dagger-android-support:$daggerVersion"
```

Comment:
I created setup that uses *dagger.android.support* with AppCompatActivity and support Fragments. Take a look at #49.